webpack/assets/javascripts/react_app/components/notifications/notifications.scss

```diff
@@ -40,21 +36,6 @@ $color-pf-blue-400: #0088ce;
       position: static;
     }
 
-    .drawer-pf-close,
-    .drawer-pf-toggle-expand {
-      color: inherit;
-      cursor: pointer;
-      line-height: inherit;
-      padding: 2px 10px;
-      position: absolute;
-
-      &:hover,
-      &:focus {
-        color: $color-pf-blue-400;
-        text-decoration: none;
-      }
-    }
-
     .drawer-pf-close {
```

Comment:
I thought that most / all of these css can go away?

Reply:
Yes, I am working on it on a different pr. It's need a bigger refactoring.